### ISSUE TITLE
Need a little bit of help regardig a lecture

### ISSUE THREAD

**Issue by JunaidHassanCB** [campusx-official/movie-recommender-system-tmdb-dataset#7]
Hi!

I needed to ask a query related to your following youtube tutorial

"Book Recommender System | Machine Learning Project | Collaborative Filtering Based Recommender Sys"

You have not mentioned anywhere about how can we check the accuracy or do evaluation on this model can you please tell me how to do this for this specific project

Thanks

**Comment by LuluW8071** [comment]
> Hi!
> 
> I needed to ask a query related to your following youtube tutorial
> 
> "Book Recommender System | Machine Learning Project | Collaborative Filtering Based Recommender Sys"
> 
> You have not mentioned anywhere about how can we check the accuracy or do evaluation on this model can you please tell me how to do this for this specific project
> 
> Thanks

I dont think you can check accuracy on `cosine_similarity` type algorithms. If it were `regression models`, `ensemble` or `neural network training` you could check accuracy. 

To check accuracy u need to split datasets into `train and test `
